addon-test-support/-private/get-element.js

```diff
@@ -18,7 +18,7 @@ export default function getElement(selectorOrElement, contextEl) {
   if (contextEl instanceof HTMLElement) {
     result = contextEl.querySelector(selectorOrElement);
   } else {
-    result = document.querySelector(`${settings.rootElement} ${selectorOrElement}`);
+    result = document.querySelector(`${settings.rootElement} ${selectorOrElement || ''}`);
```

Comment:
What do you think of adding `selectorOrElement = ''` on the function signature? It could be more self-documenting  for a likely migration to typescript? 
/cc @chriskrycho any take on what is preferable on typescript?

Reply:
@cibernox Done!

Reply:
That'll work fine with TS. You can just do this on a function signature

```ts
function foo(s: string | HTMLElement = '') {
  //...
}
```

And it'll work fine.